add a --custom arg to unify cli command (#2006)
demisto/demisto-sdk: CHANGELOG.md

* Added support for new XSIAM entities in **create-content-artifacts** command.
* Added support for Parsing/Modeling Rule content item in the **unify** command.
* Added the integration name, the commands name and the script name to the known words by default when running the **doc-review** command.

> * Added a flag '-c' '--custom' to the **unify** command, if True will append to the unified yml name/display/id the custom label provided

Reviewer: ```suggestion
* Added an argument '-c' '--custom' to the **unify** command, if True will append to the unified yml name/display/id the custom label provided
```
